js/modules/k6/experimental/fs/module_test.go

```diff
@@ -0,0 +1,695 @@
+package fs
+
+import (
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/afero"
```

Comment:
It will be preferable if `afero` is not directly used in tests as well. 

If there is a need more things can be added to fsext - but given that it is already possible for other modules, I expect you only need to use more of hte package.

See https://github.com/grafana/k6/pull/3025 for usage and changes across the codebase.